client/layout/masterbar/logged-in.jsx

```diff
@@ -281,6 +288,16 @@ class MasterbarLoggedIn extends Component {
 							{ translate( 'Write' ) }
 						</AsyncLoad>
 					) }
+					{ shouldShowCartIcon && (
+						<AsyncLoad
+							require="./masterbar-cart-wrapper"
+							placeholder={ null }
+							className="masterbar__item-cart"
+							tooltip={ translate( 'View my Shopping Cart' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **35** times:
`translate( 'My shopping cart' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/calypso-masterbar-cart)